src/commands/bam.rs

```diff
@@ -0,0 +1,23 @@
+use super::is_in_incorrect_channel;
+use serenity::{
+    client::Context,
+    framework::standard::{macros::command, CommandResult},
+    model::channel::Message,
+};
+
+#[command]
+#[aliases("kick", "ban")]
+#[description("Fake bans a user.")]
+#[usage("")]
+#[example = ""]
+#[help_available]
+#[bucket = "potentially_big_output_ever_channel"]
+pub(crate) async fn bam(ctx: &Context, msg: &Message) -> CommandResult {
```

Comment:
I don't see an use for this one. It feels like all it does is trolling, which we don't want in the server.

Reply:
hm okay.

Reply:
deleted.

Reply:
thanks :)